src/sentry/static/sentry/app/components/dropdownAutoComplete.jsx

```diff
@@ -11,6 +11,7 @@ class DropdownAutoComplete extends React.Component {
 
   static defaultProps = {
     isOpen: false,
+    alignMenu: 'left',
```

Comment:
nit: I'm guessing more of our controls will be right aligned but I have no data to back that up